README.md

```diff
@@ -210,6 +211,31 @@ bonfire deploy advisor --set-image-tag quay.io/cloudservices/advisor-backend=my_
 * `--optional-deps-method <hybrid|all|none>` -- change the way that bonfire processes ClowdApp optional dependencies (see "Dependency Processing" section)
 * `--prefer PARAM_NAME=PARAM_VALUE` -- in cases where bonfire finds more than one deployment target, use this to set the parameter names and values that should be used to select a "preferred" deployment target. This option can be passed in multiple times. `bonfire` will select the target with the highest amount of "preferred parameters" on it. Default is currently set to `ENV_NAME=frontends` to select "stable" frontends in the consoledot environments.
 
+## Trusted/Untrusted Resource Configurations
+
+When templates are processed, bonfire removes all untrusted CPU/Memory requests and limits from ClowdApp/ClowdJob/ClowdJobInvocation objects within OpenShift templates (this is because the default value for `--remove-resources` is `all`). If the resource config is unset in a ClowdApp, Clowder will set the deployments to use default values defined in the “ClowdEnvironment” to take effect.
+
+ The reason for this behavior is because many app templates tend to request far more CPU/memory in their template compared to what they actually need in a test environment.
+
+You can use the `--no-remove-resources` option to turn this behavior off for certain apps or components but the preferred approach is to audit your CPU/memory configurations and set up a resource configuration that bonfire trusts.
+
+Once you have audited your container CPU/mem usage in test environments and have determined efficient values that you want to use, you can indicate that bonfire should trust the CPU/Memory requests and limits by adhering to the following requirements:
+
+1. Any ClowdApp/ClowdJob/ClowdJobInvocation in your template asking for CPU/mem using requests or limits should use a parameter with a name formatted like this:
+* `CPU_REQUEST_<NAME>`
```

Comment:
I think if I was coming into this documentation blind I'd be a bit confused by this. I wouldn't know if `<NAME>` had to be some specific or consistent token, or if it could be arbitrary. Reading the code I know its any arbitrary combination of uppercase letters, numbers, and underscores, but reading this doc I wouldn't get that. Also, could be worth calling out what will pass the regex format - something like "where `<NAME>` is any text you want that is conforms to ... ". Finally, an example would be best. Some people will read it as written and do it right and get it, some will ask for clarification I think.